specification/azurestackhci/resource-manager/Microsoft.AzureStackHCI/stable/2023-09-01/logicalNetworks.json

```diff
@@ -0,0 +1,629 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "AzureStackHCI",
+    "description": "The Microsoft.AzureStackHCI Rest API spec.",
+    "version": "2023-09-01",
+    "x-ms-code-generation-settings": {
+      "name": "AzureStackHCIClient"
+    }
```

Comment:
The x-ms-code-generation-setting extension is being deprecated. Please remove it and move settings to readme file for code generation.
Location: Microsoft.AzureStackHCI/stable/2023-09-01/logicalNetworks.json#L7

Reply:
I'm not sure if this is really a blocker, seems like you were already flagged for this with the old api version. Any hints, @raych1 ?

Reply:
fixed